feat: lat-long popup is draggable [GEOCODE-100]

kswenson opened the pull request:
[[GEOCODE-100](https://concord-consortium.atlassian.net/browse/GEOCODE-100)] Lat/Long point is draggable

- Lat/long point is draggable
- lat/long popup shrinks when moving and expands when static

tealefristoe reviewed (approved):
Looks good 👍 

I noticed a couple of things while playing around with this.
- If you drag by clicking on the dot, the cursor stays as a pointer while dragging instead of grabbing.
- It would be really helpful to have a hover state for the copy button to make it very clear whether clicking will drag or copy.

kswenson commented:
> I noticed a couple of things while playing around with this.
> 
> * If you drag by clicking on the dot, the cursor stays as a pointer while dragging instead of grabbing.
> * It would be really helpful to have a hover state for the copy button to make it very clear whether clicking will drag or copy.

Added a commit which addressed these issues as well as comments Chris Lore made on [[GEOCODE-36](https://concord-consortium.atlassian.net/browse/GEOCODE-36)]:
- Lat/Long mode stays active until Lat/Long buttons is clicked again (rather than until it is placed)
- Lat/Long display is cleared when Lat/Long mode is exited

And finally, also makes the `Key` button clickable (it was being overlaid by the progress bar)

kswenson commented:
@tealefristoe Requesting re-review of the latest commit since the changes were more extensive than those requested and addressed some additional issues.

kswenson commented on src/components/lava-coder/lat-long-popup.tsx:
```diff
@@ -59,55 +104,107 @@ export function LatLongPopup({ viewer, verticalExaggeration, mode }: IProps) {
       window.removeEventListener("pointermove", handlePointerMove, true);
     };
-  }, [viewer]);
+  }, [setLatLongFromScreenPos, viewer]);
```
`setLatLongFromScreenPos` is defined with `useCallback` with dependencies `verticalExaggeration` and `viewer`, both of which are generally stable for a given run of the application.
